repository.rosinstall

```diff
@@ -31,10 +31,14 @@
 - git:
     local-name: lma_kinematics_plugin
     uri: 'https://github.com/shadow-robot/lma_kinematics_plugin'
```

Comment:
Can we test it without this source dependency?
I think this package is released for kinetic.
As far as i remember the changes we made in this ik_with_joint _limits branch are not very important, so it's worth testing the original repo.